docs/processes/release/tasks/integration.md

```diff
@@ -0,0 +1,428 @@
+---
+layout: default
+parent: Release & Milestone Tasks
+grand_parent: Contributors
+nav_order: 60
+---
+
+<!--
+© 2021 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Integration Tests
+{: .no_toc }
+
+## Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Run ICU4J Locale Service Provider tests
+
+JDK 6 introduced the locale service provider interface. ICU4J release include a
+special jar file (icu4j-localespi.jar) implementing the locale service provider
+interface.
+
+The test case for the ICU4J locale service provider requires JRE 6 or later
+version. Otherwise, there is no extra settings except for the standard ant set
+up ( necessary. To run the test cases, use ant with the top-level build.xml with
+target "localespiCheck"
+
+$ ant localespiCheck
+
+You should get the output like below -
+
+...
```

Comment:
code block

currently runs into a big sausage, see https://hugovdm.github.io/icu/processes/release/tasks/integration.html

Reply:
Done - desausageified, various other fixes to this page's markdown.

Text with orange background doesn't look as nice here: I can do a <span>, but not a <p>, as the latter breaks markdown.

Reply:
Great that you figured out how to do the orange background... but (in the future) if it's too onerous, it should be ok to use simpler means of emphasis, such as italics or bold.

Reply:
Agreed: just didn't want to think too much about it for this conversion, and trusting future editors to evaluate the presentation. :)